python-checks/src/test/resources/checks/unreadPrivateMethods.py

```diff
@@ -0,0 +1,28 @@
+class A:
+  __foo = 42 # OK, raised by S4487
+
+  def __unused(self): ... # Noncompliant {{Remove this unused class-private '__unused' method.}}
```

Comment:
Super nitpicky but is there a reason why we name the method specifically in the message, yet we don't do it for classes in S3985?

Reply:
Good question. @nicolas-harraudeau-sonarsource , wdyt?

Reply:
For the record, I decided to edit S3985 message to include class name.